tests/components/androidtv/test_media_player.py

```diff
@@ -144,20 +158,9 @@ async def _test_reconnect(hass, caplog, config):
     with patchers.patch_connect(True)[patch_key], patchers.patch_shell("1")[
         patch_key
     ], patchers.PATCH_ANDROIDTV_OPEN, patchers.PATCH_SIGNER:
-        # Update 1 will reconnect
         await hass.helpers.entity_component.async_update_entity(entity_id)
 
-        # If using an ADB server, the state will get updated; otherwise, the
-        # state will be the last known state
-        state = hass.states.get(entity_id)
-        if patch_key == "server":
-            assert state.state == STATE_STANDBY
-        else:
-            assert state.state == STATE_OFF
-
-        # Update 2 will update the state, regardless of which ADB connection
-        # method is used
-        await hass.helpers.entity_component.async_update_entity(entity_id)
+        # await hass.helpers.entity_component.async_update_entity(entity_id)
```

Comment:
Please remove commented code. Thank you

Reply:
Fixed in https://github.com/home-assistant/core/commit/787003c7ee35a93e87a94e547552334aed187edd